web/ui/module/codemirror-promql/README.md

```diff
@@ -245,12 +250,6 @@ Note: In case this parameter is provided, then the rest of the configuration is
 * [ReactJS example](https://github.com/prometheus/prometheus/blob/431ea75a11ca165dad9dd5d629b3cf975f4c186b/web/ui/react-app/src/pages/graph/CMExpressionInput.tsx)
 * [Angular example](https://github.com/perses/perses/blob/28b3bdac88b0ed7a4602f9c91106442eafcb6c34/internal/api/front/perses/src/app/project/prometheusrule/promql-editor/promql-editor.component.ts)
 
-## Contributions
-
-Any contribution or suggestion would be really appreciated. Feel free
-to [file an issue](https://github.com/prometheus-community/codemirror-promql/issues)
-or [send a pull request](https://github.com/prometheus-community/codemirror-promql/pulls).
-
 ## License
 
-[MIT](./LICENSE)
+[Apache](./LICENSE)
```

Comment:
I'd add the version like in https://github.com/prometheus/prometheus/blob/main/README.md#license.